### PR TITLE
Using Systemd (Config changes + rolling upgrades)

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Summary of changes:
- Added the `advertise-host` flag to systemd configuration files and parameters table
- Updated the rolling upgrades process for systemd

Tested on Digital Ocean.

### CODE REVIEW

File: _includes/v2.0/prod-deployment/secure-start-nodes.md [hunk 1]
```diff
@@ -138,6 +138,7 @@ For each initial node of your cluster, complete the following steps:
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```
Comment (jseldess) [inline _includes/v2.0/prod-deployment/secure-start-nodes.md]: I'm in the process of updating this and other text around `--advertise-host` in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.

File: _includes/v2.0/prod-deployment/insecure-start-nodes.md [hunk 2]
```diff
@@ -130,6 +130,7 @@ For each initial node of your cluster, complete the following steps:
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```
Comment (jseldess) [inline _includes/v2.0/prod-deployment/insecure-start-nodes.md]: I'm in the process of updating this and other text around --advertise-host in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.

File: _includes/v2.1/prod-deployment/insecure-start-nodes.md [hunk 3]
```diff
@@ -121,15 +121,14 @@ For each initial node of your cluster, complete the following steps:
...
-    Save the file in the `/etc/systemd/system/` directory
-
 8. Customize the sample configuration template for your deployment:
 
      Specify values for the following flags in the sample configuration template:
 
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```
Comment (jseldess) [inline _includes/v2.1/prod-deployment/insecure-start-nodes.md]: I'm in the process of updating this and other text around --advertise-host in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.

File: _includes/v2.1/prod-deployment/secure-start-nodes.md [hunk 4]
```diff
@@ -129,15 +129,14 @@ For each initial node of your cluster, complete the following steps:
...
-    Save the file in the `/etc/systemd/system/` directory.
-
 9.  Customize the sample configuration template for your deployment:
 
     Specify values for the following flags in the sample configuration template:
 
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```
Comment (jseldess) [inline _includes/v2.1/prod-deployment/secure-start-nodes.md]: I'm in the process of updating this and other text around --advertise-host in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.

File: _includes/v2.0/prod-deployment/secure-start-nodes.md (hunk 1 above)
Comment (Amruta-Ranade) [reply _includes/v2.0/prod-deployment/secure-start-nodes.md]: IIRC, we decided that I will include the --advertise-host description for the systemd section, and you will update the manual process. Does your PR take care of both? If yes, I will happily revert my change.

Comment (jseldess) [reply _includes/v2.0/prod-deployment/secure-start-nodes.md]: Yes, my PR will cover both, so you can revert this.

**Pull request merged**